### PR TITLE
fix: add herkura

### LINKED ISSUE #1674: Hershil Piplani
### Profile screenshot

Github Readme

### PR DESCRIPTION
**Pull request by herkura**
resolves #1674 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/herkura.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/54028832/214070601-70731901-12c3-4223-a6ba-4851d3e4437d.png",
```
Comment (Panquesito7) [inline profiles/herkura.json:2]: I recommend more of an image like this. What do you think?

image

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/214107857-e96feb1f-eadb-49e5-8246-2f95b291293b.png",
```

Comment (herkura) [reply profiles/herkura.json:2]: Well I thought that I should cover all the aspects of my GitHub read me profile. Hence provided that image. If there is a problem then I will use the image you provided and create another PR.

Comment (herkura) [reply profiles/herkura.json:2]: What do you say?

Comment (Panquesito7) [reply profiles/herkura.json:2]: The thing is that the image is way too big. We encourage having medium-sized photos. 🙂

Comment (herkura) [reply profiles/herkura.json:2]: Sure, then I am fine with the image you have provided

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by herkura** [comment]
No problem, anytime!

**Comment by herkura** [comment]
@Panquesito7 when will the pr get merged?

**Review by himanshu1221 — APPROVED** [review]
LGTM :+1:

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**